hilti/src/rt/tests/stream.cc

```diff
@@ -320,10 +322,10 @@ TEST_CASE("Block iteration") {
     CHECK(block->is_last);
     CHECK(! v.nextBlock(block));
 
-    x.append("567");
-    x.append("890");
-    x.append("abc");
-    x.append("def");
+    x.append(Bytes::Base("567"));
```

Comment:
Is `Bytes::Base` considered part of the public API now? Or would one normally use the `"xxx"_b` constructor instead?

Reply:
This was an oversight, using a `Bytes` literal is definitely the recommended approach. Re: interface, I'd argue that `Base` was already leaked into the public interface before (as a useful indirection, though). A possible future change to hide it fully could be to e.g., move away from inheriting from `string` to a `private` member; this should be easy now as with this change `Bytes` isn't a `string` anymore as far as users are concerned.

Reply:
Yeah, hiding that in the future sounds right.